projects/canopy/src/lib/feature-toggle/feature-toggle.directive.ts

```diff
@@ -61,8 +61,6 @@ export class LgFeatureToggleDirective implements OnInit, OnDestroy {
 
   ngOnDestroy() {
     /* istanbul ignore else */
```

Comment:
Just wondering, do we need this line anymore?

Reply:
Removed this comment.